src/sentry/static/sentry/app/components/dropdownAutoComplete.jsx

```diff
@@ -1,204 +1,40 @@
-        };
-      })
-      .filter(group => group.items.length > 0);
-
-  autoCompleteFilter = (items, inputValue) => {
-    let itemCount = 0;
-
-    if (items[0] && items[0].items) {
-      //if the first item has children, we assume it is a group
-      return _.flatMap(this.filterGroupedItems(items, inputValue), item => {
-        return [
-          {...item.group, groupLabel: true},
-          ...item.items.map(groupedItem => ({...groupedItem, index: itemCount++})),
-        ];
-      });
-    } else {
-      return this.filterItems(items, inputValue).map((item, index) => ({...item, index}));
-    }
-  };
-
   render() {
+    let {children, ...props} = this.props;
+
     return (
-      <div style={{position: 'relative', display: 'inline-block'}}>
-        {this.state.isOpen && (
-          <StyledMenu>
-            <AutoComplete itemToString={item => item.searchKey} onSelect={this.onSelect}>
-              {({
-                getRootProps,
-                getInputProps,
-                getMenuProps,
-                getItemProps,
-                inputValue,
-                selectedItem,
-                highlightedIndex,
-                isOpen,
-              }) => {
-                return (
-                  <div {...getRootProps()}>
-                    <StyledInputContainer>
-                      <StyledInput
-                        autoFocus
-                        {...getInputProps({onBlur: this.toggleMenu})}
-                      />
-                    </StyledInputContainer>
-                    <div {...getMenuProps()}>
-                      <div className="ref-autocomplete-items">
-                        {this.autoCompleteFilter(this.props.items, inputValue).map(
-                          (item, index) =>
-                            item.groupLabel ? (
-                              <StyledLabel key={item.value}>{item.label}</StyledLabel>
-                            ) : (
-                              <StyledItem
-                                key={item.value}
-                                highlightedIndex={highlightedIndex}
-                                index={item.index}
-                                {...getItemProps({item, index: item.index})}
-                              >
-                                {item.label}
-                              </StyledItem>
-                            )
-                        )}
-                      </div>
-                    </div>
-                  </div>
-                );
-              }}
-            </AutoComplete>
-            {this.props.action && (
-              <StyledActionContainer>{this.props.action}</StyledActionContainer>
-            )}
-          </StyledMenu>
-        )}
-        <div onClick={this.openMenu}>
-          {this.props.children({
-            isOpen: this.state.isOpen,
-            selectedItem: this.state.selectedItem,
-          })}
-        </div>
-      </div>
+      <DropdownAutoCompleteMenu {...props}>
```

Comment:
This is so we can re-use `DropdownAutoCompleteMenu` with different "actors" (e.g. Dropdown uses a button, but in breadcrumbs we have it open the dropdown on hover)